File: 151002/Poluectov/Rv-project/discussion/src/main/java/com/poluectov/reproject/discussion/model/Message.java
package com.poluectov.reproject.discussion.model;


import edu.umd.cs.findbugs.annotations.DefaultAnnotation;
import jakarta.persistence.Entity;
import lombok.*;
import org.springframework.data.cassandra.core.cql.Ordering;
import org.springframework.data.cassandra.core.cql.PrimaryKeyType;
import org.springframework.stereotype.Service;
import org.springframework.data.cassandra.core.mapping.PrimaryKeyColumn;
import org.springframework.data.cassandra.core.mapping.Table;
import java.math.BigInteger;
import java.util.UUID;


@Table(value = "tbl_message")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class Message extends IdentifiedEntity{


    @PrimaryKeyColumn(name = "country",
            type = PrimaryKeyType.PARTITIONED)
    private String country;

    @PrimaryKeyColumn(name = "issueid",
            ordinal = 0,
            type = PrimaryKeyType.CLUSTERED)
    private Long issueId;

    @PrimaryKeyColumn(
            name = "id",
            ordinal = 1,
            type = PrimaryKeyType.CLUSTERED)
    private Long id;

    private String content;
}

File: 151003/Denisenko/Discussion/MessageEntity/Message.cs
using Discussion.Common;

namespace Discussion.MessageEntity
{
    public class Message(string content, string country) : AbstractEntity
    {
        public string Country { get; set; } = country;
        public int IssueId { get; set; }
        public string Content { get; set; } = content;

        public Message() : this(string.Empty, string.Empty) { }

        public Message(int id, int issueId, string content) : this(content, string.Empty)
        {
            Id = id;
            IssueId = issueId;
        }

        public Message(int id, int issueId, string content, string country) : this(content, country)
        {
            Id = id;
            IssueId = issueId;
        }
    }
}
